libkbfs/data_types.go

```diff
@@ -941,3 +941,8 @@ func SessionInfoFromProtocol(session keybase1.Session) (SessionInfo, error) {
 		VerifyingKey:   verifyingKey,
 	}, nil
 }
+
+// NodeMetadata has metadata about a node needed for higher level operations.
+type NodeMetadata struct {
```

Comment:
As long as we don't end with people complaining when the contents changes. Done.